src/pages/Facility/settings/locations/components/LocationChildren.tsx

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from "@tanstack/react-query";
+import React from "react";
+
+import CareIcon from "@/CAREUI/icons/CareIcon";
+
+import { TableCell, TableRow } from "@/components/ui/table";
+
+import query from "@/Utils/request/query";
+import {
+  LocationList as LocationListType,
+  getLocationFormLabel,
+} from "@/types/location/location";
+import locationApi from "@/types/location/locationApi";
+
+interface Props {
+  facilityId: string;
+  location: LocationListType;
+  level: number;
+}
+
+export function LocationChildren({ facilityId, location, level }: Props) {
+  const paddingClasses = ["pl-4", "pl-8", "pl-12", "pl-16", "pl-20"];
```

Comment:
this is not required if we are doing recursive rendering right? what if there are more levels?